test/IDE/rdar38248403.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-ide-test -reconstruct-type -source-filename %s | %FileCheck %s
```

Comment:
Could you give this file a more descriptive name instead of the radar id?

Reply:
In fact I would just put this test case in the reconstruct_type_from_mangled_name

Reply:
Moved to the other file.